components/learn/ExternalRecommendedReadings.vue

```diff
@@ -21,8 +24,15 @@
           :segment="link.segment"
           :url="link.url"
         >
-          {{ link.label }}
+          <span v-if="link.author">
+            {{ link.author }}
+          </span>
+          <span :class="link.author ? 'external-recommended-readings__title' : ''">
+            {{ link.label }}
+          </span>
         </AppLink>
+        <span v-if="link.description">-</span>
+        {{ link.description }}
```

Comment:
I think this should go within the `<span>` tags unless you saw an error that I might've missed.